qlib/backtest/high_performance_ds.py

```diff
@@ -308,7 +311,7 @@ def __init__(self, data):
         self.data = data
```

Comment:
I think we should remove the `data` parameter of `__init__`  from `BaseOrderIndicator` and it's  subclasses.